docs/examples/fd_fitting_intermediate/index.rst

```diff
@@ -0,0 +1,374 @@
+contour length changes for these experiments. To this end, we take the model we just fitted and determine
+a contour length per data point of this model, while keeping all other parameters the same.
+
+Let's load the data and have a look::
+
+    dynamic_file = pylake.File("RecA/20200430-182304 FD Curve 40.h5")
+    dynamic_name, dynamic_curve = dynamic_file.fdcurves.popitem()
+    dynamic_curve.plot_scatter()
+
+.. image:: output_10_6.png
+
+Once again, we extract our data up to 25 pN. We can reuse the function we defined earlier::
+
+    force_dynamic, distance_dynamic = extract_data(dynamic_curve, 0, 25)
+
+A contour length per point
+--------------------------
+
+Now comes the more challenging part. Inverting the model for contour length. Luckily, this
+procedure has already been implemented in Pylake. The function `parameter_trace` inverts the
+model for a particular model parameter. Let's have a look at the parameters it needs::
+
+    >>> help(pylake.parameter_trace)
+
+    Help on function parameter_trace in module lumicks.pylake.fitting.detail.parameter_trace:
+
+    parameter_trace(model, parameters, inverted_parameter, independent, dependent, **kwargs)
+        Invert a model with respect to one parameter. This function fits a unique parameter for every data point in
+        this data-set while keeping all other parameters fixed. This can be used to for example invert the model with
+        respect to the contour length or some other parameter.
+
+        Parameters
+        ----------
+        model : Model
+            Fitting model.
+        parameters : Parameters
+            Model parameters.
+        inverted_parameter : str
+            Parameter to invert.
+        independent : array_like
+            vector of values for the independent variable
+        dependent: array_like
+            vector of values for the dependent variable
+        **kwargs:
+            forwarded to scipy.optimize.least_squares
+
+        Examples
+        --------
+        ::
+            # Define the model to be fitted
+            model = pylake.inverted_odijk("model") + pylake.force_offset("model")
+
+            # Fit the overall model first
+            data_handle = model.add_data("dataset1", f=force_data, d=distance_data)
+            current_fit = pylake.Fit(model)
+            current_fit.fit()
+
+            # Calculate a per data point contour length
+            lcs = parameter_trace(model, current_fit[data_handle], "model/Lc", distance, force)
+
+Let's see if we have all these pieces of information. Our model was called `m_dna`. We can extract
+the parameters for the RecA condition using the name we provided to the dataset before (i.e. `fit["RecA"]`).
+The parameter we wish to invert for is `DNA/Lc` and for the independent and dependent variables we simply
+pass the dataset::
+
+    Lcs = pylake.parameter_trace(m_dna, fit["RecA"], 'DNA/Lc', distance_dynamic, force_dynamic)
```

Comment:
It would be good to keep the string style consistent `""` vs `''`. My preference is usually `""` since it makes it easier to manage moving between Python and C++, but either is fine as long as it's used consistently throughout the docs. Don't worry about updating everything. Right now, it would just be good to have a single style on a single line of code :)

Reply:
Whoops. Not sure how that even happened. 😊